utils/graph.h

```diff
@@ -21,6 +21,7 @@ struct uftrace_graph_node {
 	int nr_edges;
 	int nr_calls;
 	struct graph_time_stat total_time;
+	struct graph_time_stat self_time;
 	uint64_t child_time;
```

Comment:
I think you can remove `child_time` now.

Reply:
Is `child_time` not required to get the `self_time`?

Reply:
It should come from fstack.

Reply:
@namhyung I have no idea can you explain more?

Reply:
In `add_graph_exit()` you already used `fstack->total_time - fstack->child_time` to calculate the self time.  So `node->child_time` is not needed anymore.

Reply:
@namhyung @honggyukim 

How should I modify the code below when I remove `child_time` from the `utrace_graph_node` structure?

```
...
if (sample && flame->sample_time)
		sample = (node->time - node->child_time) / flame->sample_time;
...
```
https://github.com/namhyung/uftrace/blob/master/cmds/dump.c#L1075-L1084